controls/pcidss_4.yml

```diff
@@ -1,2042 +1,2625 @@
       - base
-    status: partial
+    status: automated
     controls:
     - id: 6.3.1
-      title: 'Security vulnerabilities are identified and managed'
+      title: Security vulnerabilities are identified and managed
       levels:
         - base
-      status: pending
-      rules: []
+      status: manual
 
     - id: 6.3.2
-      title: 'An inventory of bespoke and custom software, and third-party software
-        components incorporated into bespoke and custom software is maintained to
-        facilitate vulnerability and patch management.Known vulnerabilities in
-        third-party software components cannot be exploited in bespoke and custom
-        software.This requirement is a best practice until 31 March 2025, after which it
-        will be required and must be fully considered during a PCI DSS assessment.'
+      title: An inventory of bespoke and custom software, and third-party software components
+        incorporated into bespoke and custom software is maintained to facilitate vulnerability
+        and patch management.
+      description: |-
+        Known vulnerabilities in third-party software components cannot be exploited in bespoke
+        and custom software.This requirement is a best practice until 31 March 2025, after which
+        it will be required and must be fully considered during a PCI DSS assessment.
       levels:
         - base
-      status: pending
-      rules: []
+      status: manual
+      notes: |-
+        This requirement is a best practice until 31 March 2025, after which it will be required
+        and must be fully considered during a PCI DSS assessment.
 
     - id: 6.3.3
-      title: 'All system components are protected from known vulnerabilities by
-        installing applicable security patches/updates'
+      title: All system components are protected from known vulnerabilities by installing
+        applicable security patches/updates.
+      description: |-
+        All system components are protected from known vulnerabilities by installing
+        applicable security patches/updates as follows:
+        - Critical or high-security patches/updates (identified according to the risk ranking
+        process at Requirement 6.3.1) are installed within one month of release.
+        - All other applicable security patches/updates are installed within an appropriate time
+        frame as determined by the entity (for example, within three months of release).
       levels:
         - base
       status: automated
       rules:
+        - dconf_db_up_to_date
         - ensure_suse_gpgkey_installed
         - ensure_gpgcheck_globally_activated
-        - dconf_db_up_to_date
         - ensure_gpgcheck_never_disabled
         - security_patches_up_to_date
 
-  - id: 6.4
-    title: 'Public-facing web applications are protected against attacks.'
+  - id: '6.4'
+    title: Public-facing web applications are protected against attacks.
     levels:
       - base
     status: pending
```

Comment:
I think the status can be changed to "manual" as all subsections are also manual... What do you think? It holds for other 6.x sections.

Reply:
Good catch. I found some other similar cases. Fixed in fd78237a15e160e6752595ec1599f37b9a13d3ab